src/main/java/hudson/plugins/plot/PlotAction.java

```diff
@@ -59,7 +59,12 @@ public boolean hasPlots() throws IOException {
     }
 
     // called from PlotAction/index.jelly
-    public Job<?, ?> getProject() {
+    public AbstractProject<?, ?> getProject() {
```

Comment:
`@Deprecated` (and to avoid spamming comments, suggest doing the same in the other places where you have a bridge API)